src/main/java/swervelib/motors/TalonFXSwerve.java

```diff
@@ -408,7 +408,9 @@ public void setCurrentLimit(int currentLimit) {
 		cfg.refresh(configuration.CurrentLimits);
 		cfg.apply(
 				configuration.CurrentLimits.withStatorCurrentLimit(currentLimit)
-						.withStatorCurrentLimitEnable(true));
+						.withStatorCurrentLimitEnable(true)
+						.withSupplyCurrentLimit(80)
+						.withSupplyCurrentLimitEnable(true));
```

Comment:
@jbko6 - this really ought to be configurable so it can be different between the drive & steer motors.